next.config.js

```diff
@@ -47,7 +47,7 @@ module.exports = withBundleAnalyzer(
             headers: securityHeaders,
           },
           {
-            source: '/themes/(.*)',
+            source: '/(themes/(?:.*)|client\\.js)',
```

Comment:
Is this regex correct? :thinking:

Shouldn't it be `\.js` instead of `\\.js`? I'm not a regex expert, though.

```suggestion
            source: '/(themes/(?:.*)|client\.js)',
```